lib/commands/smapi/smapi-command-handler.js

```diff
@@ -115,8 +116,13 @@ const _showCheckStatusHint = (locationHeader) => {
         const baseCliCommand = processor.processOperationName(operationName);
         const queryStringParameters = queryString.parse(queryStringPart);
         parameters = { ...swaggerMatch.params, ...queryStringParameters };
-        const cliParams = Object.keys(parameters).map(key => `--${kebabCase(key)} ${parameters[key]}`).join(' ');
-        statusCheckHintCommand = `ask smapi ${baseCliCommand} ${cliParams}`;
+        const cliParams = Object.keys(parameters).map(key => {
+            // mapping custom command option if exists
+            const commandOption = kebabCase(apiToCommanderMap.get(key) || key);
```

Comment:
main use case. if parameter is called stageV2 it is called stage in ask cli
https://github.com/alexa/ask-cli/blob/develop/lib/commands/smapi/customizations/parameters.json#L3